_quarto.yml

```
@@ -7,6 +7,10 @@ project:
 book:
   title: Git for Collaborative Teams
   subtitle: "An introductory course on team-based collaboration with Git and GitHub"
+  body-header: |
+    ::: {.callout-warning icon="true"}
```

```suggestion
    ::: {.callout-warning appearance="default" icon="true"}
```

The rostools theme changes the appearance to minimal, so this ensures it uses the default look.